Kernel/Interrupts/GenericInterruptHandler.h

```diff
@@ -48,6 +48,8 @@ class GenericInterruptHandler {
 
     virtual bool eoi() = 0;
     void increment_call_count();
+    void set_reserved() { m_reserved = true; };
+    bool is_reserved() const { return m_reserved; };
```

Comment:
This should just be `reserved()` - getters should reflect the variable name according to our coding style